resources/guides/ta_guide.md

```diff
@@ -0,0 +1,99 @@
+---
+id: teaching assistant
+title: How To Become a Teaching Assistant
+sidebar_label: How To Become a Teaching Assistant
```

Comment:
Sidebar space is a precious commodity. I suggest an alternate label, "Becoming a TA" or "Becoming a Teaching Assistant" to save space.